browser/ui/webui/brave_wallet/android/android_wallet_page_ui.cc

```diff
@@ -77,6 +77,12 @@ AndroidWalletPageUI::AndroidWalletPageUI(content::WebUI* web_ui,
       std::string("frame-src ") + kUntrustedTrezorURL + " " +
           kUntrustedLedgerURL + " " + kUntrustedNftURL + " " +
           kUntrustedLineChartURL + " " + kUntrustedMarketURL + ";");
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::ImgSrc,
+      base::JoinString({"img-src", "'self'", "chrome://resources",
+                        "chrome://erc-token-images", "chrome://favicon",
+                        "chrome://image", base::StrCat({"data:", ";"})},
+                       " "));
```

Comment:
why is this last line needed?

also it seems the default is `chrome://favicon2` now instead of `chrome://favicon` but probably we don't need either of them

Reply:
cc: @samartnik

Reply:
The last line is the second param (`separator`) for `base::JoinString`. `chrome://favicon` and `chrome://image` are not required, removed them here https://github.com/brave/brave-core/commit/13ebf4e7b805514e624d5e3d3c16a4c7c66d4a41

Reply:
ah sorry i missed that it wasn't part of the first parameter. lgtm.